src/graphql/root/mutation/account-data-update.ts

```diff
@@ -0,0 +1,60 @@
+import { getAccountsConfig } from "@config"
+
+import { Accounts } from "@app"
+
+import { GT } from "@graphql/index"
+import { mapError } from "@graphql/error-map"
+import { parseCustomFieldsSchema } from "@graphql/helpers"
+import AccountDataUpdatePayload from "@graphql/types/payload/account-data"
+
+const customFieldsSchema = getAccountsConfig().customFields
+
+const AccountDataUpdateInput = GT.Input({
+  name: "AccountDataUpdateInput",
+  fields: () => {
+    const customFields = parseCustomFieldsSchema(customFieldsSchema)
+
+    return {
+      transactionsCallback: { type: GT.String },
+      ...customFields,
```

Comment:
Since certain custom fields are required, will they always have to be passed to update any AccountData?

Reply:
yes (also an independent record is persisted in db every time the data is sent), if we want a different behavior we will need to add a mutation for `creation` and other to `update` 
@nicolasburtey any preference here?